include/unicorn/arm.h

```diff
@@ -24,6 +24,9 @@ typedef enum uc_arm_reg {
 	UC_ARM_REG_FPSCR_NZCV,
 	UC_ARM_REG_FPSID,
 	UC_ARM_REG_ITSTATE,
+	UC_ARM_REG_C1_C0_2,
+	UC_ARM_REG_C13_C0_2,
```

Comment:
did you add this `UC_ARM_REG_C13_C0_2`, but never used that in `qemu/target-arm/unicorn_arm.c` ?